SWLOR.Game.Server/Service/AbilityService/RecastGroup.cs

```diff
@@ -177,6 +177,12 @@ public enum RecastGroup
         Rejuvenation = 84,
         [RecastGroup("Frenzied Shout", "Frenz. Shout")]
         FrenziedShout = 85,
+        [RecastGroup("Force Death", "Force Death")]
+        ForceDeath = 85,     
```

Comment:
Force Death is reusing ID 85 which is incorrect. Please increment this and all subsequent IDs by 1 to avoid overlapping with Frenzied Shout.